### PR TITLE
Add sunburst documentation

### LINKED ISSUE #1294: sunburst docs
- [ ] javascript
- [ ] python
- [ ] r

### PR DESCRIPTION
**Pull request by michaelbabyn**
resolves #1294

### CODE REVIEW

File: _posts/plotly_js/basic/sunburst/2019-04-11-sunburst_plotly_js_index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Javascript Graphing Library D3.js-based Sunburst Plots | Examples | Plotly
+name: Sunburst Plots
+permalink: javascript/sunburst/
```
Comment (cldougl) [inline _posts/plotly_js/basic/sunburst/2019-04-11-sunburst_plotly_js_index.html]: let's make the permalink `javascript/sunburst-charts/`

File: _posts/plotly_js/basic/sunburst/2019-04-11-sunburst_plotly_js_index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Javascript Graphing Library D3.js-based Sunburst Plots | Examples | Plotly
+name: Sunburst Plots
```
Comment (cldougl) [inline _posts/plotly_js/basic/sunburst/2019-04-11-sunburst_plotly_js_index.html]: just a suggestion but I think the terms Sunburst Charts or Sunburst Graphs are a bit more common

File: _posts/r/basic/2019-04-12-sunburst-chart.Rmd [hunk 3]
```diff
@@ -0,0 +1,145 @@
+---
+title: R Sunburst Charts | Examples | Plotly
+name: Sunburst Charts
+permalink: r/sunburst/
```
Comment (cldougl) [inline _posts/r/basic/2019-04-12-sunburst-chart.Rmd]: sunburst-charts

File: _posts/python/basic/sunburst/2015-06-30-sunburst-charts.html [hunk 4]
```diff
@@ -0,0 +1,292 @@
+---
+permalink: python/sunburst/
```
Comment (cldougl) [inline _posts/python/basic/sunburst/2015-06-30-sunburst-charts.html]: sunburst-charts

File: _includes/documentation_eg.html [hunk 5]
```diff
@@ -627,6 +627,14 @@ <h4 class="example-section-title" id="layout-options">
 </div>
 {% endif %}
 
+{% if page.dash_url %}
```
Comment (cldougl) [inline _includes/documentation_eg.html]: was this codeblock added on purpose to this pr?

Comment (michaelbabyn) [reply _includes/documentation_eg.html]: No, just a dirty repo... I'm getting rid of it.

**Pull request merged**